x-pack/filebeat/input/azureeventhub/config.go

```diff
@@ -121,7 +149,8 @@ func (conf *azureInputConfig) Validate() error {
 	}
 
 	// log a warning for each sanitization option not supported
-	for _, opt := range conf.SanitizeOptions {
+	for _, opt := range conf.LegacySanitizeOptions {
+		logger.Warnw("legacy sanitization `sanitize_options` options are deprecated and will be removed in future releases; use the `sanitizers` option instead", "option", opt)
```

Comment:
Should we also mark in the documentation about the `sanitize_options` config will be removed in the future? Maybe specify in 9.0?

Reply:
+1 on this suggestion :) we should define the removal version and create a follow-up ticket so we don't forget about this :)

Reply:
9.0 is handy for deprecation purposes; I'm creating a note to track this.